docker/docker-compose.yml

```diff
@@ -0,0 +1,21 @@
+version: "0.1"
+name: cwydsa
+services:
+  web:
+    image: fruoccopublic.azurecr.io/rag-webapp
+    ports:
+      - "8080:80"
+    env_file:
+      - ../.env
```

Comment:
Is this going to work with the files that are generated by azd?

Reply:
Not out of the box no. We could try and use the `AZURE_ENV_NAME` env var, but I don't think there are guarantees that will be set. Alternatively, we could have a script wrapper around this that calls `azd env list`, gets the default env, and passes that to docker compose e.g. `docker-compose --env-file .my-env  up`. WDYT?

Reply:
Irrespective of whether people have Windows/Linux/DevContainers, then they are going to need a Search Service/OpenAi to communicate with. There is a line in the MakeFile (at the top) that will get you the correct .env file - maybe we should wrap the docker-compose in the MakeFile

Reply:
Good point, I'll see what can be done with the Makefile 👍